docs/ingressroute.md

```diff
@@ -553,6 +553,43 @@ spec:
           port: 80
 ```
 
+#### Stick Sessions Support
+
+Route-level HashPolicy combined with Service-level hash-based load balancing is used to support sticky sessions.
+
+```yaml
+apiVersion: contour.heptio.com/v1beta1
+kind: IngressRoute
+metadata:
+  name: app
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: app.example.com
+  routes:
+    - match: /
+      hashPolicy:
+        # Precisely one of header, cookie, or connectionProperties must be set
+      - connectionProperties:
+          sourceIp: true
+        # Precisely one of header, cookie, or connectionProperties must be set
+        cookie:
+          name: some-cookie
+          path: cookie/path
+          ttl: 60s
```

Comment:
If I pass TTL as a go duration string such as `60s`, the `hashPolicy` for my route doesn't get set.

However, if I set it as an int such as `60`, then it does get set, but in nanoseconds: `0.000000060s`. I believe that this is because the `ttl` value hasn't been parsed by `time.ParseDuration` before being used to construct the  `HashPolicyCookie`.

I also noticed that the cookie TTL doesn't appear in the tests, so we should add it.

Reply:
I'll add the parsing support and additional tests. I'm not seeing tests that parse the CRD which would have caught this